### PR TITLE
Add memory-pair-game

### PR DESCRIPTION
**Pull request by Vitaminvp**
### Memory Pair Game
[Demo](https://vitaminvp.github.io/kottans-frontend/task-Memory-Pair-Game) |
[Code base](https://github.com/Vitaminvp/kottans-frontend/tree/master/task-Memory-Pair-Game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Vitaminvp/memory-pair-game/app.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                if (flipContainer.classList.contains('on')) {
+                    flipContainer.classList.remove('on');
+                } else {
+                    flipContainer.classList.add('on');
+                    this.callback(true, imageSrc, flipContainer.id);
+                    setTimeout(() => {
+                        flipContainer.classList.remove('on');
+                        this.callback(false);
+                        this.id = "";
+                    }, 2000);
```
Comment (zonzujiro) [inline submissions/Vitaminvp/memory-pair-game/app.js]: Magic numbers are [bad](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

File: submissions/Vitaminvp/memory-pair-game/app.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            this.renderItems();
+        });
+    }
+
+    renderItems() {
+        this.output.innerHTML = "";
+        const docFragment = document.createDocumentFragment();
+        
+        const arrOfImg = APP_CONFIG.arrImgSrc.slice(0, APP_CONFIG.width);
+        //const totalBlocksAmount = arrOfImg.reduce((acc, cur, i, arr) => [...acc, ...shuffle([...arr])], []);
```
Comment (zonzujiro) [inline submissions/Vitaminvp/memory-pair-game/app.js]: Pls, remove commented code from PR

File: submissions/Vitaminvp/memory-pair-game/app.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        const totalBlocksAmount = shuffle(arrOfImg.reduce((acc, cur, i, arr) => [...acc, ...arr], []));
+        for (let i = 0; i < BLOCKS_AMOUNT; i++) {
+            new Card(docFragment, this.countInc.bind(this), totalBlocksAmount[i]);
```
Comment (zonzujiro) [inline submissions/Vitaminvp/memory-pair-game/app.js]: You can use arrow function instead of `bind`

**Comment by Vitaminvp** [comment]
Ok. Please, check again.

**Comment by zonzujiro** [comment]
@Vitaminvp pls, resolve outdated discussions

**Pull request merged**